lib/dialects/abstract/index.js

```diff
@@ -11,6 +11,7 @@ AbstractDialect.prototype.supports = {
   'ORDER NULLS': false,
   'UNION': true,
   'UNION ALL': true,
+  'CITEXT': false,
```

Comment:
No need for this in `supports`, these flags are meant to switch code segments when generating queries or declaring support information that is used anywhere in `lib/`, for tests you can just compare `dialect === 'postgres'`